analyses/infercnv-consensus-cell-type/project-workflows/run-SCPCP000015.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This script runs the module workflow for SCPCP000015 and is meant to be called from ./run-analysis.sh.
+# It can be run as follows, optionally specifying the following variables:
+#
+# testing=<0 or 1> threads=<number of threads for inferCNV> seed=<random seed to set for inferCNV> ./run-SCPCP000015.sh
+
+set -euo pipefail
+
+# Ensure script is being run from _one directory up_, which represents the root directory of the module
+# This is so `renv` loads properly, without needing to call `renv::load()` in scripts since code run in Docker containers won't use `renv`
+module_dir=$(dirname "${BASH_SOURCE[0]}")/..
+cd ${module_dir}
```

Comment:
I am fine with this approach and I think ultimately it will help keep the root directory less cluttered.